cfme/cloud/instance.py

```diff
@@ -1,736 +0,0 @@
-""" A model of Instances page in CFME."""
```

Comment:
This was removed in favor of an instance package, base class and multi-class methods are now in instance/__init__.py.